Guide/Constructive/New Year Chaos/README.md

```diff
@@ -0,0 +1,102 @@
+# New Year Chaos
+This problem comes from [HackerRank](https://www.hackerrank.com/challenges/new-year-chaos)
+and is categorized as a Constructive Algorithm
+
+HackerRank Difficulty: Medium
+
+
+## Problem
+### Statement

```

Comment:
Should we say that this is referenced from Hackerrank? I would actually suggest replacing this section with a link to the problem statement in Hackerrank or a short explanation by you. Their TOS actually says we can't use content from their site directly.